firebase-firestore/src/roboUtil/java/com/google/firebase/firestore/ValueHelper.java

```diff
@@ -0,0 +1,98 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.firestore;
+
+import com.google.firebase.Timestamp;
+import com.google.firebase.firestore.model.DatabaseId;
+import com.google.firebase.firestore.model.DocumentKey;
+import com.google.firestore.v1.ArrayValue;
+import com.google.firestore.v1.Value;
+import com.google.protobuf.NullValue;
+import com.google.type.LatLng;
+import java.util.List;
+
+/** Test helper to create Firestore Value protos from Java types. */
+public class ValueHelper {
+
+  public static Value valueOf(Object o) {
+    if (o instanceof Value) {
+      return (Value) o;
+    } else if (o instanceof String) {
+      return (Value.newBuilder().setStringValue((String) o).build());
```

Comment:
Extra parens, here and throughout.

Reply:
Removed. Sorry, I was quite certain `googleJavaFormat` would do this for me.